### PR TITLE
Generalize a few probes and make them use SQLAlchemy

### PR DESCRIPTION
**Pull request by ericvaandering**
Supercedes #27 

What do you think @dchristidis ?

### CODE REVIEW

**Comment by ericvaandering** [comment]
Remove fast_count references when more recent Rucio is available.

**Comment by dchristidis** [comment]
My sincerest apologies for not taking note of this. Would you mind rebasing against the current master? Additionally, please add yourself as an author.

**Comment by ericvaandering** [comment]
OK @dchristidis this is now done from my side. Rebased onto the new master with @tbeerman changes. 

Please merge. I verified it works.  One last issue is the probes container has an old version of Rucio in it for some reason so the function I'm relying on is not there and is put into each of these.

File: common/check_new_dids [hunk 1]
```diff
@@ -37,8 +46,8 @@ if __name__ == "__main__":
     try:
         registry = CollectorRegistry()
         session = get_session()
-        new_dids = 'select  /*+ index_FFS(dids DIDS_IS_NEW_IDX) */   count(1) from {schema}dids where is_new is not null'.format(schema=schema)
```
Comment (dchristidis) [inline common/check_new_dids:40]: I don’t have DB expertise, but I see no necessity to remove this hint for Oracle.

Comment (ericvaandering) [reply common/check_new_dids:40]: Can I add hints in SQLAlchemy?

Comment (ericvaandering) [reply common/check_new_dids:40]: Never mind, I found it. It doesn't seem to make a timing difference but it's included now

**Pull request merged**